### PR TITLE
Augury [Intro] Page — improve grammar, phrasing, and fix spelling mistakes

### PR DESCRIPTION
**Pull request by adrianmcli**
I have gone through the Augury [intro] page and have made a bunch of changes to:

* Spelling;
* Grammar, and;
* Phrasing for ease of understanding.

Note that I have also changed all references to keyboard keys so that they use the native `<kbd>` tag for better styling flexibility and to be in compliance with documentation best practices.

Example: <kbd>ctrl</kbd> instead of `ctrl`.

### CODE REVIEW

File: guide/augury/augury.md [hunk 1]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
 * Events
 * Object properties
 
-Augury compliments DevTools during a debugging session, making it easy to modify states and emit events.
+Augury compliments DevTools during a debugging session, making it easy to modify state and emit events.
 
 ## Installing Augury
 
-The best way to install Augury is to head over to [chrome web store](https://chrome.google.com/webstore/category/extensions?hl=en). Once there, select _Extensions_ from the side panel and type "Augury" into the search field, then press _Enter_.
+The best way to install Augury is from the [chrome web store](https://chrome.google.com/webstore/category/extensions?hl=en). Select _Extensions_ from the side panel, type "Augury" into the search field, and then press <kbd>Enter</kbd>.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Let's change `<kbd>` back to indent, I already wrote a guideline and don't want styles changed.

File: guide/augury/augury.md [hunk 2]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
 
 ![Image Extension logo](images/extension-logo.png)
 
-The Augury icon provides additional information. Click on the icon now to discover what that is.
+The Augury icon provides additional information. Click on the icon now to see what is available.
 
 ## Using Augury
 
-To start using Augury, you must have an Angular application running in the browser for inspection. If you have never debugged a JavaScript application, you may not be aware that each modern Web browser provides a debug environment straight in the browser. _DevTools_, the debug environment is opened using the following _shortcut_:
+To start using Augury, you must have an Angular application running in the browser. If you have never debugged a JavaScript application before, you may not be aware of the debugging environment that most modern browsers provide.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Let's revert the "Using Augury" Section to what is was before, it intros new style plus this is copied through out the other guide and I would rather not make changes.

File: guide/augury/augury.md [hunk 3]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
 
 When DevTools is opened, you will find the Augury tab on the far right.
 
 ![Image DevTools](images/devtools.png)
 
-## Augury features
+## Augury Features
 
-We will walk quickly through the main functionality that is available in Augury. This is to become familiar with the features and how to locate them when needed.
+We will quickly go through the main functionality available in Augury. This will allow you to become familiar with the features so you can locate them when needed.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: I am using the active voice so no use of, he, she, me you. Please revert.

File: guide/augury/augury.md [hunk 4]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
+The search result should list the Augury extension by _Rangle.io_:
 
 ![Image Augury extension](images/augury-extension.png)
 
 When you click on "Add To Chrome", a _popup_ will open. Select "Add extension" to complete the process. Once the _plugin_ has been successfully installed, an Augury icon will appear next to the address bar in the browser.
 
 ![Image Extension logo](images/extension-logo.png)
 
-The Augury icon provides additional information. Click on the icon now to discover what that is.
+The Augury icon provides additional information. Click on the icon now to see what is available.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Please revet.

File: guide/augury/augury.md [hunk 5]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
+  CSS should use `max-width: 100%` for all img tags -->
 <img src="images/properties.png" style="width: 100%">
 
 Notable items of interest are:
 
-1. View source link
-1. Change Detection in use
-1. Object properties
-1. Dependencies
+1. **View Source** — a link to the source code of the component.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: I like the addition details, just remove the bold to keep inline with style.

File: guide/augury/augury.md [hunk 6]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
+
+* For Windows and Linux, use <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>I</kbd>
+* For Mac OS X, use <kbd>Cmd</kbd> + <kbd>Opt</kbd> + <kbd>I</kbd>
 
 When DevTools is opened, you will find the Augury tab on the far right.
 
 ![Image DevTools](images/devtools.png)
 
-## Augury features
+## Augury Features
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Please revet all headings to how they were, this is the style I choose. See guide like I slacked for details.

File: guide/augury/augury.md [hunk 7]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
+We will quickly go through the main functionality available in Augury. This will allow you to become familiar with the features so you can locate them when needed.
 
 The first view that is visible is the _Component Tree_ which shows **loaded** components belonging to the application.
 
 ![Image Component tree](images/component-tree.png)
 
-The component tree shows the hierarchal relationship of the _components_. When a _component_ is selected, Augury presents additional information about the _component_ in the _Properties_ tab.
+The component tree displays a hierarchical relationship of the _components_. When a _component_ is selected, Augury presents additional information about the selected _component_ in the _Properties_ tab.
 
+<!-- instead of using HTML here, the global
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: remove comment, I don't have control over this, the website is compiled and not going to dig into the code :)

File: guide/augury/augury.md [hunk 8]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
-1. Dependencies
+1. **View Source** — a link to the source code of the component.
+1. **Change Detection** — displays whether or not Change Detection is in use for the component.
+1. **Object Properties** — lists the properties of the component.
+1. **Dependencies** - lists the dependencies of the component.
 
 To view the source code of the selected _component_, click the 'View Source' link. This will bring the _Sources_ tab into focus and display the source code.
 
-### Source map
+### Source Map
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: revert

File: guide/augury/augury.md [hunk 9]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
 
-One thing to keep in mind, the _TypeScript_ code will only be shown if a _source map_ from the build exists. In production if no _source map_ is available, what you will see is the compiled JavaScript code, which may also be minified and difficult to read.
+One thing to keep in mind is that the _TypeScript_ code will only be shown if a _source map_ file exists. In production, if no _source map_ is found, what you will see instead is the compiled JavaScript code, which may also be minified and difficult to read.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: 1. "is that the" sounds off to me, just revet this only
1. remove "instead", no alternative is presented, we're stating a fact not comparing.

File: guide/augury/augury.md [hunk 10]
```diff
@@ -16,71 +16,74 @@ Augury provides better insight to an Angular application structure and the relat
...
 
-Next to the _Properties_ tab is the _Injector Graph_, clicking on it will display the dependency of _components_ and _services_.
+Next to the _Properties_ tab is the _Injector Graph_, clicking on it will display the dependency relationships of _components_ and _services_.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Adding an extra word don't really do much, my rule is to write less :)

File: guide/augury/augury.md (hunk 2 above)
Comment (adrianmcli) [inline guide/augury/augury.md]: I didn't delete the sentence showing how to open devtools, I decided to put the final sentence on its own line, but I can revert it if you like.

File: guide/augury/augury.md [hunk 11]
```diff
@@ -20,7 +20,7 @@ Augury compliments DevTools during a debugging session, making it easy to modify
 
 ## Installing Augury
 
-The best way to install Augury is from the [chrome web store](https://chrome.google.com/webstore/category/extensions?hl=en). Select _Extensions_ from the side panel, type "Augury" into the search field, and then press <kbd>Enter</kbd>.
+The best way to install Augury is from the [chrome web store](https://chrome.google.com/webstore/category/extensions?hl=en). Select _Extensions_ from the side panel, type "Augury" into the search field, and then press `Enter`.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: Enter should be indented, only code is backticked.

File: guide/augury/augury.md [hunk 12]
```diff
@@ -32,49 +32,45 @@ When you click on "Add To Chrome", a _popup_ will open. Select "Add extension" t
...
 
 When DevTools is opened, you will find the Augury tab on the far right.
 
 ![Image DevTools](images/devtools.png)
 
-## Augury Features
+## Augury features
 
-We will quickly go through the main functionality available in Augury. This will allow you to become familiar with the features so you can locate them when needed.
+We will walk quickly through the main functionality that is available in Augury. This is to become familiar with the features and how to locate them when needed.
```
Comment (rajinder-yadav) [inline guide/augury/augury.md]: "Walk quickly" is a grammatically odd phrase for technically writing, we're not actually walking, the other changes are fine. Let's change it to, "quickly go over".

File: guide/augury/augury.md (hunk 11 above)
Comment (adrianmcli) [inline guide/augury/augury.md]: @rajinder-yadav By indented, do you mean italicized with the underscores (as it originally was)?

Also, isn't it inconsistent that the key groups (see line 41) use back ticks, but this key does not? The style guide states that "code artifacts" are to be surrounded by back ticks. Key groups are not code artifacts.

File: guide/augury/augury.md (hunk 12 above)
Comment (adrianmcli) [inline guide/augury/augury.md]: This was the original line, I simply reverted back to it as per your instructions. I can make the change.

**Comment by rajinder-yadav** [comment]
:shipit:

**Pull request merged**